velox/connectors/hive/HiveConfig.h

```diff
@@ -35,10 +35,14 @@ class HiveConfig {
   static constexpr const char* kMaxPartitionsPerWriters =
       "max_partitions_per_writers";
 
+  static constexpr const char* kReadAsLowerCase = "read_as_lower_case";
+
   static InsertExistingPartitionsBehavior insertExistingPartitionsBehavior(
       const Config* config);
 
   static uint32_t maxPartitionsPerWriters(const Config* config);
+
+  static bool isReadAsLowerCase(const Config* config);
```

Comment:
Would you document this configuration property here and in https://facebookincubator.github.io/velox/configs.html ?